### ISSUE TITLE
JOSS/review/paper

### ISSUE THREAD

**Issue by JannisHoch** [hydroframe/subsettools#85]
Dear authors,

thanks for the work on this package (very relevant) and submitting a paper to JOSS.
Overall, the paper is well written and contains all relevant section.
I especially applaud that the package was already tested in practice outside academia.

Here a few comments which should be addressed before publication:

- [ ] Statement of need section: are the files you collected to solve barrier 2 also usable for commercial parties, i.e., under which license can they be used? 
- [ ] Statement of need section: what is CONUS1 and CONUS2?
- [ ] State of the field: how does this package compare to similar packages? This is not discussed. If there are not similar commonly-used packages, maybe worth stating this fact.
- [ ] A more general remark: this package seems to be focussing on US applications. What are your thoughts and plans on expanding this to the global scale?

Many thanks for addressing these comments!

**Comment by gartavanis** [comment]
Thank you, @JannisHoch, for taking the time to review our paper and for your comments. We will address each of your individual concerns below. 

1. Statement of need section: are the files you collected to solve barrier 2 also usable for commercial parties, i.e., under which license can they be used?

The whole package is covered under the MIT license and we have added a sentence to the start of the functionality section to make this more clear.

2. Statement of need section: what is CONUS1 and CONUS2?

We have clarified the definitions of CONUS1 and CONUS2 in the Summary section by moving up our definition of CONUS (the contiguous United States) and changing the wording from first and second generation to CONUS1 and CONUS2. Also, both words are hyperlinked to pages describing the attributes of the CONUS1 and CONUS2 domains.

3. State of the field: how does this package compare to similar packages? This is not discussed. If there are not similar commonly-used packages, maybe worth stating this fact.

SubsetTools is the only package of its kind that accesses and processes input data and then builds a ParFlow model in one complete workflow. We have added a sentence to the end of the summary section to reflect this. While there may be other packages that interface with other hydrologic models (for example pyGSFLOW), SubsetTools is the only package that works with ParFlow. As our target users are those who work with ParFlow models, we feel it is sufficient to note that no other package exists of this type for ParFlow.

4. A more general remark: this package seems to be focussing on US applications. What are your thoughts and plans on expanding this to the global scale?

We have added a new future work section that includes new features that are in production as well as addressing what it would take to expand to a global scale. If we are provided the required data for regions outside of the United States to host in our data repository, we can expand the package to work with global data.

Thank you again for your comments and we hope we have addressed all the concerns above to your satisfaction.
